qb/func.go

```diff
@@ -49,3 +49,12 @@ func MaxTimeuuid(name string) *Func {
 func Now() *Func {
 	return Fn("now")
 }
+
+// Count produces COUNT(field_name) or COUNT(*).
+func Count(paramNames ...string) *Func {
+	name := "COUNT"
```

Comment:
In other places we use lowercase function names, can we be consistent on that?

Reply:
OK.